.github/workflows/client.yml

```diff
@@ -48,13 +52,15 @@ jobs:
         with:
           node-version: '10.x'
       - run: npm install -g @ionic/cli
-      - run: npm install
-        working-directory: ./client/ionic
-      - run: ionic build
+      - run: npm ci
         working-directory: ./client/ionic
-      - run: ionic cap sync
+      - uses: actions/download-artifact@v1
+        with:
+          name: build
+          path: ./client/ionic/build
+      - run: npx capacitor sync
```

Comment:
`ionic cap sync` runs both `ionic build` and `capacitor sync`. We've already gotten the results of the former through the former so we only need to run the latter.